server/app/controllers/procedure.controller.js

```diff
@@ -90,30 +90,26 @@ const search = async (req, res) => {
   }
 };
 
-const updateClientCpt = async (req, res) => {
+const updateClientProcedure = async (req, res) => {
   const errors = validationResult(req);
   if (!errors.isEmpty()) {
     errorMessage.message = errors.array();
     return res.status(status.error).send(errorMessage);
   }
   const db = makeDb(configuration, res);
-  const { cptId, favorite, billable, fee, notes } = req.body;
+  const { procedureId, favorite, billable, fee, notes } = req.body;
   let $sql;
   try {
-    $sql = `insert into client_cpt (client_id, user_id, cpt_id, favorite, billable, fee, notes, created, created_user_id, updated, updated_user_id )
-        values (${req.client_id}, ${
-      req.user_id
-    }, '${cptId}', ${favorite}, ${billable}, ${
-      fee > 0 ? fee : 0
-    } /*TODO if fee is "" then set fee to null*/, '${notes}', now(), ${
-      req.user_id
-    }, now(), ${req.user_id} ) 
+    $sql = `insert into client_procedure (client_id, user_id, procedure_id, favorite, billable, fee, notes, created, created_user_id, updated, updated_user_id )
+        values (${req.client_id}, ${req.user_id
+      }, '${procedureId}', ${favorite}, ${billable}, ${fee > 0 ? fee : 0
+      } /*TODO if fee is "" then set fee to null*/, '${notes}', now(), ${req.user_id
+      }, now(), ${req.user_id} ) 
          on duplicate key update 
             favorite=${favorite},
             billable=${billable},
-            fee=${
-              fee > 0 ? fee : 0
-            },  /*TODO if fee is "" then set fee to null*/
+            fee=${fee > 0 ? fee : 0
+      },  /*TODO if fee is "" then set fee to null*/
```

Comment:
Still `TODO` should be here?

Reply:
I have worked on the renaming of **cpt** to **procedure**. I am not sure about the functionality of this method. We can update the logic after discussing it with @davidjhp .